### ISSUE TITLE
Adding aria-labels for accessibility Paging chevron buttons

### ISSUE THREAD

**Issue by jmannDev43** [DevExpress/devextreme-reactive#1897]
- [X] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

### I'm using ...

- [X] React Grid
- [ ] React Chart
- [ ] React Scheduler

### Description
Is it possible to add aria-labels to the PagingPanel chevron buttons?  I'm looking into using DevExtreme React Grid, but these items fail our accessibility audit.  

Thanks in advance!

### Environment

- devextreme-reactive: 1.10.0
- react: 16.8.0
- browser: Chrome 72.0.3626.121 
- bootstrap: none
- react-bootstrap: none
- material-ui: 3.9.2
